### PR TITLE
Dean updating seo section

### PR DESCRIPTION
**Pull request by dino475**
I updated the SEO section of global settings to explain that you can opt out of search engine indexing. Please feel free to edit as necessary - I'm not 100% confident this is formatted correctly.

### CODE REVIEW

File: settings/global.mdx [hunk 1]
```diff
@@ -607,10 +607,15 @@ Example:
     <ResponseField name="indexHiddenPages" type="boolean" default="false">
     Enables indexing pages not included in `navigation`.
     </ResponseField>
+    <ResponseField name="noindex" type="boolean" default="false">
+    Disables search engine indexing in your given MDX file.
+    </ResponseField>
```
Comment (mayankshouche) [inline settings/global.mdx:612]: i don't believe this exists in the mintconfig today, iirc you have to specify `noindex` per-page in the frontmatter?

**Comment by dino475** [comment]
Got it, thanks Mayank! So you're saying this doesn't belong in the mint.json? Where is the appropriate place to mention this in the docs?

File: settings/global.mdx (hunk 1 above)
Comment (rpmccarter) [reply settings/global.mdx:612]: yep mayank is correct here

**Comment by rpmccarter** [comment]
`noindex` setting does not exist in `mint.json`

**Pull request closed without merging**